pkg/client/unversioned/clientcmd/loader.go

```
@@ -136,10 +137,25 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 
 	envVarFiles := os.Getenv(RecommendedConfigPathEnvVar)
 	if len(envVarFiles) != 0 {
-		chain = append(chain, filepath.SplitList(envVarFiles)...)
-
+		files := filepath.SplitList(envVarFiles)
+		for ix := range files {
+			moreFiles, err := loadFileOrDir(files[ix])
+			if err != nil {
+				glog.Warningf("Failed to load config %s (%v)", files[ix], err)
```

The signature doesn't include an error, seems better than silently dropping?